### ISSUE TITLE
the key "`" (on the left of key 1) not work

### ISSUE THREAD

**Issue by rmGFW** [debauchee/barrier#1837]
### What happened?

the server is macos apple sillicon m1 pro , 
and the client is arch linux 

i found the "`" key not work

### Version

v2.4.0

### Git commit hash (if applicable)

2.4.0-release-3e0d758b

### If applicable, where did you install Barrier from?

homebrew

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### What OS versions are you using?

server:macos Monterey 12.6

client: archlinux ,kernel 6.0.8-arch1-1

### Relevant log output

```shell
i already check that , there is no log when i press down the "`" both in the server and client side
```

**Comment by tkdave** [comment]
Possible duplicate of #1531

**Comment by dgentry** [comment]
Yes, this is exactly #1531 and #1407 again.  There are two pending pull requests that would fix it, but which haven't been taken nor commented on.  One is PR #1711 (mine) and the other is #1541.

**Comment by jmatsushita** [comment]
Please note there is a MacOS build with @dgentry's fix [available as an Azure pipeline published artifact](https://dev.azure.com/debauchee/Barrier/_build/results?buildId=782&view=artifacts&pathAsName=false&type=publishedArtifacts).
